src/app/workspace/results/results.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute, Params, Router } from '@angular/router';
 import { SearchParams } from '@dasch-swiss/dsp-ui';
+import {ReadProject} from '@dasch-swiss/dsp-js';
```

Comment:
Missing spaces between curly brackets

https://docs.google.com/document/d/1SIVMxoV0N8xYqhVk_0hhuqSuUF47D1YYrDESqdmipfM/edit#heading=h.fwfa70355egx
and 
https://angular.io/guide/styleguide#angular-coding-style-guide

Reply:
Done